app/views/collections/show.html.slim

```diff
@@ -26,6 +28,13 @@
             =< t('.no_invites')
           - else
             =< t('.num_of_invites', count: @num_of_invites)
+        span
+          - if @collection.user_favorites.length == 0
```

Comment:
```suggestion
          - if @collection.user_favorites.length.zero?
```

Nice syntactic sugar and supporting understanding (comparison of numbers vs. "presence" check). Same as mentioned in the other MR before. ;)

More high-level question: Why making a difference here? You are using `count`, so the locale for zero favorites can easily cover the `.no_favorites` locale while the `fa-*` class can be set dynamically based on the condition.

Reply:
You are completetly right. I also refactored the "invitations" and "other users" to use `count` correctly